packages/astro/src/jsx/renderer.ts

```diff
@@ -1,19 +1,11 @@
-const renderer = {
+import type { AstroRenderer } from '../@types/astro.js';
+import { jsxTransformOptions } from './transform-options.js';
+
+const renderer: AstroRenderer = {
 	name: 'astro:jsx',
 	serverEntrypoint: 'astro/jsx/server.js',
 	jsxImportSource: 'astro',
-	jsxTransformOptions: async () => {
```

Comment:
This function is moved to `./transform-options.js` without changes. This allows using the `jsxTransformOptions` function directly.